addons/fleet/views/fleet_vehicle_model_views.xml

```diff
@@ -101,7 +101,7 @@
         <field name="name">fleet.vehicle.model.tree</field>
         <field name="model">fleet.vehicle.model</field>
         <field name="arch" type="xml">
-            <tree string="Models">
+            <tree string="Models" multi_edit="1">
```

Comment:
With your change all fields are made editable through the multi-edit. Also the field is still not directly editable when you click on the checkbox of the field. With this change (in enterprise) instead, it should do the trick:
```
<field name="can_be_requested" optional="show" widget="boolean_toggle"/>
```

Reply:
But if gmf thinks it's ok, it's ok for me :)